spark/src/test/java/org/apache/iceberg/spark/source/TestPartitionValues.java

```diff
@@ -53,11 +62,28 @@
     };
   }
 
-  private static final Schema SCHEMA = new Schema(
+  protected static final Schema SUPPORTED_PRIMITIVES = new Schema(
```

Comment:
nit: Why protected? Do we expect `TestPartitionValues` to be extended and used in another package?

Reply:
I copied this from an abstract test class. I'll fix it.